lib/gc/ExecutionEngine/CPURuntime/Microkernel/BrgemmOnednn.cpp

```diff
@@ -56,81 +57,77 @@ static std::shared_mutex g_brgemm_lock;
 struct brgemm_cache_info_t {
   brgemm_desc_t desc;
   brgemm_kernel_t *kernel;
-  std::shared_ptr<char[]> palette;
+  std::unique_ptr<char[]> palette;
 };
 
-static std::vector<brgemm_cache_info_t> g_cache;
+static std::vector<brgemm_cache_info_t> g_cache(DEFAULT_KERNEL_SIZE);
+static int64_t kernel_id = -1;
```

Comment:
maybe chanege to `g_kernel_id`? this var has the same name with some func parameters and could cause confusion,

Reply:
thanks for the comment, fixed.